src/eradiate/scenes/biosphere/_leaf_cloud.py

```diff
@@ -31,7 +31,7 @@ def _inversebeta(mu, nu, rng):
         s2 = np.power(rands[1], 1.0 / nu)
         s = s1 + s2
         if s <= 1:
-            return s1 / s
+            return s2 / s * np.pi / 2.0
```

Comment:
This is not a 2π factor: I assume the approximate was also wrong? If this departs from what's written in the reference you cite in the docstring, can you also update it?

Reply:
To be honest, I do not understand how this code follows from the cited source anyways. This is just copied from raytran. If we really want something we can cite and refer to, we should remove this and implement the distribution function by Goel and Strebel directly.

Reply:
Okay, so how about comparing the reference (Goel and Strebel) and this, both precision- and performance-wise? Since performance matters when it comes to generating those leaf clouds, I can see how an approximate would make sense.

Reply:
the performance of sampling the Goel and Strebel function using the rejection method is faster, I already checked that (however it takes single digit number of seconds to generate 100k samples in both cases, so this seems very uncritical to me)

for the precision: well there we need a statistical test. but we could instead just drop the `_inversebeta` function and just sample the original distribution and be done with it.

Reply:
Going for the Goel & Strebel function is better IMO, especially if it's what RAMI instructs—is it?